scripts/build.py

```diff
@@ -916,6 +918,14 @@ def build_engine(self):
             print "Removing", os.environ['DM_BOB_ROOTFOLDER']
             shutil.rmtree(os.environ['DM_BOB_ROOTFOLDER'])
 
+    def build_external(self):
```

Comment:
A new function that we can now use on our nightly CI, to make sure our external builds don't break.